examples/quick-batching/CMakeLists.txt

```diff
@@ -1 +1 @@
-qml_lint(quick-batching.qml Slider.qml)
+#qml_lint(quick-batching.qml Slider.qml)
```

Comment:
Temporary porting leftover? If that doesn't work with 6 we probably should at least make it conditional on the Qt version. Repeats below.

Reply:
done. examples seem to work, so renabled.